src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -660,10 +666,10 @@ gl_sframe activity_classifier::predict_topk(gl_sframe data,
     stacked_rank = stacked_rank.stack("rank", "rank");
     result.add_column(stacked_rank["rank"], "rank");
   } else {
-    // add "exp_id" and "prediction_id" if the output_frequency is per_window
-    result.add_column(raw_preds_per_window["session_id"], "exp_id");
-    result.add_column(gl_sarray::from_sequence(0, raw_preds_per_window.size()),
-                      "prediction_id");
+    // add session ID and prediction ID if the output_frequency is per_window
```

Comment:
I think the 5.8 used "exp_id" instead?

Reply:
The 5.8 Python implementation used whatever string the user provided as the column name for the session ID. It just so happens that the test data you've been using uses "exp_id". Note that this string appears nowhere in the original Python implementation

Reply:
Oh cool!